modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -764,6 +775,17 @@ definition {
 			locator1 = "ProcessBuilderKaleoDesignerReact#SELECTED_NODE");
 	}
 
+	macro viewLanguageTranslationDropdownNotPresent {
+		Click(
+			key_translationOptionSelected = "${translationOptionSelected}",
+			locator1 = "ProcessBuilderKaleoDesignerReact#ICON_COUNTRY");
+
+		AssertElementNotPresent(
+			key_languageTranslationOption = "${languageTranslationOption}",
+			locator1 = "ProcessBuilderKaleoDesignerReact#LANGUAGE_TRANSLATION_IN_DROPDOWN",
```

Comment:
Here you could use the path that it's already created : TRANSLATION_COUNTRY_LABEL.